/*

    NOTES:
    - Exception -> Runtime
    - Syntax errors -> Compile time
    - Bugs - Unexpected results

    try {
        // code block
        if() {
            // throw new CustomExceptionName(msg);
        }
    } catch(CustomExceptionName exp) {
        // code block
    } catch(secondayException exp) {
        // code block
    } catch(Exception exp) {
        // code block
    } finally {
        // code block
    }

    class CustomExceptionName extends Exception {
        public CustomExceptionName(String msg) {
            super(msg);
        }
    }

    - Throw keyword


 */

public class _31_ExceptionAndErros {

}
